microsoft/PowerToys
# [Deps]Upgrade Windows App SDK to 1.4.1

## Summary of the Pull Request

Upgrade Windows App SDK to 1.4.1.


## Validation Steps Performed

I've built the installer and opened the following utilities to make sure they're not crashing:
- Settings
- File Locksmith
- Hosts File Editor
- Peek
- Power Rename
- Registry Preview
- Screen Ruler

Will trigger release CI to make sure it's all OK as well.
